src/operation.ts

```diff
@@ -645,6 +714,16 @@ export class Callback extends Operation {
 
     return this.schema?.description ? this.schema.description.trim() : undefined;
   }
+
+  getParameters(): Array<RMOAS.ParameterObject> {
```

Comment:
Like with the `getSummary` and `getDescription` overrides that the `Callback` class has it also needs one for parameters in order to support common parameters as callback common parameters exist in the schema above the callback (`this.parentSchema`), and not at the `this.api.paths[method]` level that they're at for standard operations.

A bit annoying but there's good test coverage in place.